lib/Runtime/Types/PathTypeHandler.h

```diff
@@ -30,6 +76,9 @@ namespace Js
         virtual BOOL IsLockable() const override { return true; }
         virtual BOOL IsSharable() const override { return true; }
 
+        static PropertyAttributes ObjectSlotAttributesToPropertyAttributes(const ObjectSlotAttributes attr) { return attr & ObjectSlotAttr_PropertyAttributesMask; }
```

Comment:
>ObjectSlotAttributesToPropertyAttributes [](start = 34, length = 40)

Could we add static asserts to ensure property attributes have the same corresponding bits in PropertyAttributes and ObjectSlotAttributes?

Reply:
Yeah, there are CompileAssert's in the cpp file.